src/Libraries/CoreNodeModels/CoreNodeModels.csproj

```diff
@@ -20,14 +20,18 @@
   <ItemGroup>
     <None Remove="CoreNodeModelsImages.resources" />
   </ItemGroup>
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net48' ">
```

Comment:
I think we should just dump this node - I don't think theres anything it can do that a code block cannot.